redaxo/src/core/lib/console/setup/run.php

```diff
@@ -447,7 +446,7 @@ static function ($login) {
 
         // ---------------------------------- last step. save config
 
-        $config['setup'] = false;
+        $config['setup'] = is_array($config['setup']) ? $config['setup'] : false;
```

Comment:
würde der vllt lesbarer werden, wenn wir die $config datenstruktur in einem objekt mappen würden?

Reply:
Hab nichts dagegen, wenn wir das in ein Objekt umwandeln.
Wobei speziell diese Zeile vermutlich nicht wirklich besser wird, weil `$config->setup` dann ja trotzdem noch array oder bool wäre.

Reply:
Aber es wäre gekapselt in der klasse und nicht über redaxo verteilt. Muss aber auch nicht sein